src/app/pages/auth/containers/enrolment-page.component.ts

```diff
@@ -57,9 +57,17 @@ export class EnrolmentPageComponent {
     private localization: LocalizationService,
     private alertService: AlertService,
     private usage: UsageService,
-    private logger: LogService
+    private logger: LogService,
+    private globalization: Globalization
   ) {
-    this.localization.update().then(lang => (this.language = lang))
+    this.auth.reset()
```

Comment:
should this be done when reset is triggered and not in constructor every time? I know enrolment page will only be shown when enrolling but this may change in the future (or may have unintended consequences for someone updating a forked version of the app)

Reply:
Yeah we do clear it before when the reset is triggered, but we ran into some issues when the reset doesn't fully clear the storage. I'll try to find an alternative without resetting in the enrolment page.